### ISSUE TITLE
Actions by the board without a meeting don't appear in the list of meeting minutes

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy-website#1629]
Whoops! Those should appear, ideally pinned to the top of the list. I missed this in #1628.

**Comment by alice-i-cecile** [comment]
Actually, I'm not sure why this is failing. Maybe because I didn't add a date??

**Comment by cart** [comment]
Yup its the date. Also worth prefixing the folder with the date to ensure it sorts chronologically in text editors (like we do for other news types)

**Comment by alice-i-cecile** [comment]
The big question here is what date to put. We could put the founding of the foundation, the last time it's updated, or maybe there's a better idea.

**Comment by alice-i-cecile** [comment]
>  I kind of think each "action without a board meeting" should be its own "meeting"

Alright, I'm on board with Cart's idea there. It needs to be chopped up, but this should be easy enough.

**Comment by cart** [comment]
We could add a brief preamble "boilerplate" to each "action without a meeting" to help contrast them with normal meetings.

**Comment by cart** [comment]
Also when we build a "news article tagging system", we could probably use that "for free" here as well. (add tags for "normal meetings" and "actions without meetings")

**Comment by alice-i-cecile** [comment]
Right, we can even add a short code for the preamble.
